_shared/project/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+concurrency:
+  group: deploy
+  cancel-in-progress: true
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    uses: ./.github/workflows/ci.yml
+  Docker_Hub:
+    name: Docker Hub
+    needs: CI
+    uses: hypothesis/workflows/.github/workflows/dockerhub.yml@main
+    with:
+      Application: {% raw %}${{ github.event.repository.name }}{% endraw +%}
+    secrets: inherit
+  {% if include_exists(".github/workflows/environments.json") %}
+  {% for job_id, environment in include_json(".github/workflows/environments.json").items() %}
```

Comment:
:+1: Done, this works.

Unfortunately I've got a feeling this won't work always work and in many cases `{% if include_exists("...") %}` may actually be required. I think we may only be getting away with it in this particular case because the entire contents of the `{% if %}` is just a `{% for %}`.